src/main/java/org/sunbird/extension/user/impl/UserProviderRegistryImpl.java

```diff
@@ -0,0 +1,80 @@
+package org.sunbird.extension.user.impl;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.http.MediaType;
+import org.sunbird.extension.user.UserExtension;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.google.gson.JsonObject;
+
+import io.opensaber.registry.client.OpensaberClient;
+import io.opensaber.registry.client.data.RequestData;
+import io.opensaber.registry.client.data.ResponseData;
+import io.opensaber.registry.exception.TransformationException;
+import io.opensaber.registry.transform.ITransformer;
+import io.opensaber.registry.transform.JsonToJsonLDTransformer;
+import io.opensaber.registry.transform.JsonldToJsonTransformer;
+
+/**
+ * Implementation Class of UserExtension Interface
```

Comment:
User profile extension using Open Saber registry for storing adopter specific custom user details.